internal/k8s/pod_template.go

```diff
@@ -0,0 +1,47 @@
+package k8s
+
+import (
+	"crypto"
+	"fmt"
+
+	"github.com/pkg/errors"
+	v1 "k8s.io/api/core/v1"
+)
+
+const TiltPodTemplateHashLabel = "tilt-pod-template-hash"
```

Comment:
maybe name the key something like
tilt.dev/pod-template-hash
per the naming conventions
https://kubernetes.io/docs/concepts/overview/working-with-objects/common-labels/